dist/profile/manifests/kubernetes/delete.pp

```diff
@@ -19,8 +19,10 @@
 #     }
 #
 define profile::kubernetes::delete (
+  String $context = '',
   String $resource = $title,
-  $clusters = $profile::kubernetes::params::clusters
+  String $user = $profile::kubernetes::params::user,
+  String $kubeconfig = $profile::kubernetes::params::kubeconfig,
 ){
   include ::stdlib
   include profile::kubernetes::params
```

Comment:
Shouldn't there be a similar `fail()` statement in this resource when `$context` == ''